### ISSUE TITLE
Rewrite to check in parallel

### ISSUE THREAD

**Issue by gaborcsardi** [r-lib/revdepcheck#2]
Features we want to have
* [ ] `docker` & `localbuilder` support (`localbuilder` needs to mature a lot before this....)
* [x] ~persistent state in `liteq`~
* [x] multiple workers via `processx` / `sys`
* [x] Nice summarization, run with old and new package, compare.
* [ ] `sysreqs` support
* [x] time limit for a check

@krlmlr ideas welcome!

**Comment by gaborcsardi** [comment]
Moved outstanding items to separate issue, this is done now.
